---
title: "Projects"
toc: true
toc-depth: 2
---

## Current work as bioinformatician at the University of Amsterdam

I am currently working as bioinformatician at the [Institute for Biodiversity and Ecosystem Dynamics (IBED)](https://ibed.uva.nl/) at the [University of Amsterdam](https://www.uva.nl/en). 

My role is to bridge the intersection of biology and computer science and to guide and support scientists, postdocs, and PhD students in the bioinformatic analysis of their data. I am doing this as member of the [IBED Computational Support Team](https://ibed.uva.nl/facilities/computational-facilities/ibed-computational-support-team/ibed-computational-support-team.html).

My areas of expertise (see also below for a more detailed description) include amplicon sequencing, metagenomics as well as phylogenomics. Additionally, I am adept in using R and Python to analyse computational data.



## Postdoc at the NIOZ (Netherlands)

<span style="font-size:1.5vw;">
**[Exploring the hidden world of DPANN archaea using metagenomic approaches]{.underline}**
</span>

As a Postdoctoral researcher at the Royal Netherlands Institute for Sea Research (NIOZ) in the Netherlands, I was working in the lab of [Dr. Anja Spang](https://www.nioz.nl/en/about/organisation/staff/anja-spang).

My research focused on using metagenomic approaches to investigate archaea, one of the main divisions of life alongside Bacteria and Eukaryotes. Archaea live in a variety of habitats on Earth, including soils, sediments, marine and freshwater environments, and even the human gut.

Through metagenomics, we can sequence DNA directly from environmental samples and reconstruct genomes from uncultivated organisms. This technique has revealed the diversity of microbial life and the roles of these organisms in their habitats. In particular, the DPANN archaeal superphylum contains a large diversity of putative symbiotic organisms, many of which have small genomes and limited metabolic capabilities, suggesting that they rely on other microbes for growth and survival. The few DPANN archaea that have been cultivated are likely obligate symbionts or parasites.

<p align="center">
<img src="images/Binning.png" width="50%"/>
<figcaption class = "center">Going from DNA to reads to contigs to genomes</figcaption>
</p>

My research focused on gaining insights into the biology and evolution of DPANN archaea. Specifically, I am interested in understanding the diversity of DPANN archaea in marine environments, developing approaches to identify the interaction partners of DPANN archaea, and finding the position of DPANN in the archaeal tree of life. This will help to address the question of whether symbiosis evolved early in the archaeal domain.

<p align="center">
<img src="images/undinarchaeota.png" width="90%"/>
<figcaption class = "center">Right: Placing the Undinarchaeota, , members of the DPANN archaea, into the archaeal tree. <br> Left: Potential dependencies of Undinarchaeota with a host as inferred from metagenomics. </figcaption>
</p>

Find out more:

- [Genomic diversity, lifestyles and evolutionary origins of DPANN archaea](https://pubmed.ncbi.nlm.nih.gov/30629179/)
- [Talking about a novel phylum in the DPANN archaea](https://www.youtube.com/watch?v=6whnZfdFZTw)

<br>

<span style="font-size:1.5vw;">
**[Investigating Microbial Diversity and Nutrient Dependencies in the Black Sea]{.underline}**
</span>

As a postdoctoral researcher at the Royal Netherlands Institute for Sea Research (NIOZ), I also investigated microbial communities in the Black Sea using metagenomic approaches. This stratified and anaerobic marine environment provides stable conditions that make it an ideal study site.

Through this research line, we aim to compare microbial communities at different depths and untangle the nutrient interdependencies of microbes in both the water column and sediment. Our preliminary data has revealed that DPANN archaea are extremely diverse and abundant in the Black Sea. As such, we are working to build methods to predict the host of these DPANN archaea based on sequencing data.

By understanding the microbial diversity and nutrient dependencies in the Black Sea, we hope to gain a better understanding of how these microorganisms interact with their environment and contribute to global biogeochemical cycles.

<p align="center">
<img src="images/IMG_3162.jpg" width="50%"/>
<figcaption class = "center">Collecting water samples from the Black Sea</figcaption>
</p>

<br>

## Postdoc at the University of Texas (USA)

<span style="font-size:1.5vw;">
**[Investigating the Diversity and Metabolic Potential of Hydrothermal Vent-Associated Microbes]{.underline}**
</span>

During my postdoctoral work at the Marine Science Institute (University of Texas) in the [lab of Dr. Brett Baker](https://sites.utexas.edu/baker-lab/) I focused on understanding the diversity and metabolic potential of microbes found in hydrothermal vent sediments. By reconstructing microbial genomes and analyzing their metabolic capabilities, I aimed to answer two key questions:

- How diverse are the microbial communities found in Guaymas Basin deep-sea hydrothermal vent sediments?
- What is the genetic, functional, and metabolic potential encoded in the bacterial genomes from these sediment communities?

By studying these communities, I aimed to gain insights into the role of microbes in hydrothermal vent ecosystems and their potential for biotechnological applications.

Find out more:

- [Expansive microbial metabolic versatility and biodiversity in dynamic Guaymas Basin hydrothermal sediments](https://www.nature.com/articles/s41467-018-07418-0)


<p align="center">
<img src="images/hydrothermal.jpg" width="50%"/>
<figcaption class = "center">Microbial mats found on hydrothermal sediments (Guaymas Basin)</figcaption>
</p>

<br>

<span style="font-size:1.5vw;">
**[Genome reconstruction of oil-degrading bacteria]{.underline}**
</span>

As part of my postdoctoral research, I conducted another project on identifying hydrocarbon degradation pathways in genomes from uncultured bacteria from water affected by the Deep Water Horizon Oil Spill. My research focused on the following questions:

- Which hydrocarbon-degrading bacteria were present during the Deepwater Horizon oil spill?
- What are the pathways employed by these bacteria for oil degradation?
- What genetic features do these bacteria possess that enable them to adapt quickly to a high oil input?

Through reconstructing their genomes, we gained insight into the metabolic capabilities of these bacteria and their potential role in oil spill remediation.

<p align="center">
<img src="images/Oil_spill.png" width="50%"/>
<figcaption class = "center">Oil mixture accumulating on the sea surface</figcaption>
</p>

Find out more:

- [Reconstructing metabolic pathways of hydrocarbon-degrading bacteria from the Deepwater Horizon oil spill](https://www.nature.com/articles/nmicrobiol201657)
- [Can we harness bacteria to help clean up future oil spills?](https://theconversation.com/can-we-harness-bacteria-to-help-clean-up-future-oil-spills-59358)


<br>

## Previous Research

### PhD

<span style="font-size:1.5vw;">
**[Investigating the Bacterial Microbiota in Plant Roots and Their Impact on Plant Growth]{.underline}**
</span>

During my doctoral research (2012-2015) in the [lab of Prof. Schulze-Lefert]((https://www.mpipz.mpg.de/schulze-lefert) ) at the Max-Planck-Institute for Plant Breeding Research in Cologne, Germany, I focused on two main research topics related to bacterial microbiota in plant roots:

1. Analyzing the structure and function of the bacterial root microbiota of *Arabidopsis thaliana* and relative species. Here, I aimed to answer:

- What is the structure of bacterial communities found in natural soil, rhizosphere, and plant roots of *Arabis alpina*, *Arabidopsis thaliana*, and *Cardamine hirsuta*?
- What factors (e.g., environment, soil type, flowering) shape the assembly of bacterial communities?

Find out more:

- [Root microbiota dynamics of perennial Arabis alpina are dependent on soil residence time but independent of flowering time](https://www.nature.com/articles/ismej2016109)
- [Quantitative divergence of the bacterial root microbiota in Arabidopsis thaliana relatives](http://www.pnas.org/content/111/2/585.abstract)

<p align="center">
<img src="images/arabis.jpg" width="30%"/>
<figcaption class = "center">Sampling <i>Arabis alpina</i> in its natural habitat, <br> the French Alps</figcaption>
</p>


2. Cultivating root-associated rhizobia and studying their plant growth-promoting properties on *Arabidopsis thaliana*. My research questions were:

- How diverse is the rhizobia population found on the roots of *Arabidopsis thaliana*?
- What is the role of root-associated rhizobia in plant growth?
- How do rhizobia affect the plant transcriptome?

<p align="center">
<img src="images/growth_assay.png" width="50%"/>
<figcaption class = "center">The effect of a rhizobium (R129_E) on root development <br> under different nutrient conditions</figcaption>
</p>

Find out more:

- [Modular Traits of the Rhizobiales Root Microbiota and Their Evolutionary Relationship with Symbiotic Rhizobia](https://pubmed.ncbi.nlm.nih.gov/30001518/)

<br>


### Masters

<span style="font-size:1.5vw;">
**[Investigating Auxin-Inducible Genes in *Arabidopsis thaliana* Using Translating Ribosomal Affinity Purification (TRAP)]{.underline}**
</span>

For my undergraduate research, I worked in the [QuintLab](https://quintlab.landw.uni-halle.de/team/) at the Leibniz Institute for Plant Biochemistry in Halle, Germany. My thesis focused on investigating auxin-inducible genes in *Arabidopsis thaliana* using the new experimental technique of translating ribosomal affinity purification (TRAP).

Specifically, I:

- Established TRAP as a new experimental technique to isolate cell-type-specific mRNA, as well as developed new cell-type-specific marker lines.
- Investigated the effect of auxin application on the expression of marker genes in different root-cell types.

